apps/generator /readme.md

```
@@ -1,13 +1,36 @@
 # Marketing Generator
 
+A Trigger.dev-based workflow for automatically generating marketing content, specifically focused on glossary entries (see e.g. [circuit breaker](https://unkey.com/glossary/api-circuit-breaker)). The system uses PlanetScale as its database with Drizzle ORM for data management.
+
+**Table of Contents**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)